data/customModules/Command.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * A Command Arg.
+ * @typedef  {Object}  Arg
+ * @property {String}  name         The name of the arg.
+ * @property {Boolean} [mand=false] Whether or not this arg is mandatory.
+ * @property {String}  [delim=' ']  The delimiting character for this arg.
+ */
+
+/**
+ * Class representing a command.
+ */
+class Command {
```

Comment:
I don't like how spaced out this constructor is. Don't do the individual JSDocs.

Reply:
I assume you mean the class jsdoc looks weird. Should it be a single line then?
```
/** Class representing a command. */
```

Reply:
I mean this https://i.imgur.com/vZFnIR0.png

Reply:
I'm going to leave it like this unless you can show me another method that actually works, the couple ways I tried don't seem to be effective.